Then/Then.swift

```diff
@@ -40,3 +40,16 @@ extension Then {
 }
 
 extension NSObject: Then {}
+
+infix operator => { associativity left precedence 200 }
```

Comment:
Since its assignment, it should have the same precedence as the equal(=) operator which is 90.